packages/api/server.js

```diff
@@ -14,11 +14,43 @@
 // limitations under the License.
 
 const express = require('express');
+const Repository = require('./src/repository');
 const app = express();
 const bodyParser = require('body-parser');
 
 app.use(bodyParser.json());
 
+app.get('/repos', async (req, res) => {
+  // TODO: Make it return more information about the repos, beyond just their names
+  var repository = new Repository(null);
```

Comment:
nit: but I suggest using `const` rather than `var`, if variables aren't going to be reassigned.